modules/luci-mod-dashboard/htdocs/luci-static/resources/view/dashboard/include/20_lan.js

```diff
@@ -45,10 +35,18 @@ return baseclass.extend({
 			E('h3', this.title)
 		]));
 
+		var container_devices = E('table', { 'class': 'table assoclist devices-info' }, [
+			E('tr', { 'class': 'tr dashboard-bg' }, [
+				E('th', { 'class': 'th nowrap' }, _('Hostname')),
+				E('th', { 'class': 'th' }, _('IP Address')),
+				E('th', { 'class': 'th' }, _('MAC')),
+			])
+		]);
+
```

Comment:
nitpick: is it necessary to move this?

Reply:
It is not necessary. It matches more closely how `30_wifi.js` does title first and then the rest and having it in the order it will land in the DOM seems more natural, but it can certainly stay where it was originally. Let me know which way you prefer it.